### PR TITLE
Flowmode

### PR DESCRIPTION
**Pull request by ightevenmckane1**
``Converting model...
Model converted successfully.

Standalone code to reproduce the issue: 

import tensorflow as tf

def my_function(x):
  return tf.add(x, 1)

model = tf.keras.models.Sequential([
  tf.keras.layers.Input(shape=(1)),
  tf.keras.layers.Dense(1, activation='relu'),
  my_function,
])

model.compile(optimizer='adam', loss='mse')

model.fit(x=[1, 2, 3], y=[2, 3, 4], epochs=10)

tflite_model = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model.experimental_generate_debug_info = True
tflite_model.target_spec.supported_ops = [
  tflite.OpsSet.TFLITE_BUILTIN,
  tflite.OpsSet.SELECT_TF_OPS,
]
tflite_model.output_format = tflite.Model.ModelFormat.TFLITE
tflite_model.experimental_enable_mlir_converter = True
tflite_model.experimental_enable_quantization = True
tflite_model.inference_input_type = tflite.TensorType.FLOAT32
tflite_model.inference_output_type = tflite.TensorType.FLOAT32
tflite_model.input_tensors[0].name = 'x'
tflite_model.output_tensors[0].name = 'y'
tflite_model.output_file = 'my_model.tflite'

tflite_model.convert()

> `print(tflite_model.output_file)`

> This code will create a TensorFlow Lite model and save it to a file called my_model.tflite.

Any other info / logs:

There are no other logs or source code that would be helpful to diagnose the problem. `

### CODE REVIEW

**Review by ightevenmckane1 — COMMENTED** [review]
OS Platform and Distribution: Linux Ubuntu 16.04
TensorFlow installed from: source
TensorFlow version: 2.5.0
Text output from tflite_convert:
Standalone code to reproduce the issue:

Code snippet
This code will create a TensorFlow Lite model and save it to a file called my_model.tflite.

Any other info / logs:

There are no other logs or source code that would be helpful to diagnose the problem.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
This is spam, it is not your work, just merging branches without looking at what they represent.
